packages/astro/src/core/errors/errors-data.ts

```diff
@@ -454,26 +454,75 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 			`[paginate()] page number param \`${paramName}\` not found in your filepath.`,
 		hint: 'Rename your file to `[page].astro` or `[...page].astro`.',
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [Assets (Experimental)](https://docs.astro.build/en/guides/assets/)
+	 * - [Image component](https://docs.astro.build/en/guides/assets/#image--astroassets)
+	 * - [Image component#alt](https://docs.astro.build/en/guides/assets/#alt-required)
+	 * @description
+	 * The `alt` property allow users of screen readers and other assistive technologies to understand what your image is supposed to represent. In order to ensure your websites are accessible, the Image component requires that an `alt` be specified.
```

Comment:
```suggestion
	 * The `alt` property allows you to provide descriptive alt text to users of screen readers and other assistive technologies. In order to ensure your images are accessible, the `Image` component requires that an `alt` be specified.
```
We can't ensure the whole SITE is accessible! :sweat_smile: